.vscode/tasks.json

```diff
@@ -9,6 +9,13 @@
             "-p", "tsconfig.json"
         ],
         "dependsOn": "npm: webpack-preview-dev"
+      },
+      {
+          "label": "adoc-prelaunch",
+          "dependsOn": [
+              "npm: webpack",
+              "npm: compile"
+          ]
```

Comment:
Most of the functions so far are under `package.json` and `scripts`. 

I think this is already under the `vscode:prepublish` script.

Reply:
I am not sure about this one, because I could not find documentation about prepublishing (thus why I changed the task). Do you think I should revert the change?